### PR TITLE
add vegan gnocci recipe

### PR DESCRIPTION
**Pull request by koddsson**
This was pretty chill so I wanted to have a place to keep the recipe ✨

### CODE REVIEW

File: _recipes/creamy-vegan-gnocchi-with-garlic-and-kale.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  - slug: water
+    amount:
+      - 60 ml
+      - 1/2 cup
+  - slug: gnocchi
+    amount:
+      - 500 g
+  - slug: salt
+    amount:
+      - 1/4 tps
```
Comment (keithamus) [inline _recipes/creamy-vegan-gnocchi-with-garlic-and-kale.md]: I'd like to ensure every measure has a metric measure as the first. Volumetric measurements can be problematic in cooking, especially with salt which can be very course or very fine

```suggestion 
      - 4 g
      - 1/4 tsp
```

File: _recipes/creamy-vegan-gnocchi-with-garlic-and-kale.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  - slug: freshly ground pepper
+    amount: to taste
```
Comment (keithamus) [inline _recipes/creamy-vegan-gnocchi-with-garlic-and-kale.md]: Generally speaking I'd avoid adding ingredients to taste. Either omit or provide a suitable metric measure here.

File: _recipes/creamy-vegan-gnocchi-with-garlic-and-kale.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  - slug: chili flakes
+    amount: to taste
```
Comment (keithamus) [inline _recipes/creamy-vegan-gnocchi-with-garlic-and-kale.md]: Same here - about ingredients to taste. Either omit or provide a suitable metric measure here.

File: _recipes/creamy-vegan-gnocchi-with-garlic-and-kale.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+  - pasta
+  - vegan
+prepTime:
+  - PT5M
+cookTime:
+  - PT7M
+recipeYield: 3 meals
+recipeCuisine: Italian
+recipeIngredient:
+  - slug: olive oil
```
Comment (keithamus) [inline _recipes/creamy-vegan-gnocchi-with-garlic-and-kale.md]: The slugs are used to link to pages about ingredients (which show you what other recipes use a certain ingredient). As such they should be URL slugs,

```suggestion
  - slug: olive-oil
```

File: _recipes/creamy-vegan-gnocchi-with-garlic-and-kale.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+    amount:
+      - 60 ml
+      - 1/2 cup
+  - slug: gnocchi
+    amount:
+      - 500 g
+  - slug: salt
+    amount:
+      - 1/4 tps
+  - slug: freshly ground pepper
```
Comment (keithamus) [inline _recipes/creamy-vegan-gnocchi-with-garlic-and-kale.md]: The slugs are used to link to pages about ingredients (which show you what other recipes use a certain ingredient). As such they should be URL slugs,

```suggestion
  - slug: freshly-ground-pepper
```

File: _recipes/creamy-vegan-gnocchi-with-garlic-and-kale.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+    amount: to taste
+  - slug: chili flakes
```
Comment (keithamus) [inline _recipes/creamy-vegan-gnocchi-with-garlic-and-kale.md]: The slugs are used to link to pages about ingredients (which show you what other recipes use a certain ingredient). As such they should be URL slugs,

```suggestion
  - slug: chili-flakes
```

**Comment by keithamus** [comment]
In addition, what I've done with some of the other recipes is to add sections and actions to each step - which I'm hoping will help later on with the information design around recipe steps. It might be useful to do the same here. As an example:

https://github.com/keithamus/recipes/blob/60c9867754358837763556aef95c19fea22a17f2/_recipes/thai-curry-paste.md#L83-L87

**Comment by koddsson** [comment]
Let me know what you think @keithamus ✨

**Comment by koddsson** [comment]
I submit the pictures attached as CC0 - https://creativecommons.org/choose/zero/

```
CC0 
To the extent possible under law, Kristján Oddsson has waived all copyright and related or neighboring rights to Pictures of Gnocci. This work is published from: United Kingdom.
```

**Comment by keithamus** [comment]
:+1: this looks awesome

**Pull request merged**